Documentation/DebugApi.md

```diff
@@ -1,14 +1,13 @@
 # amun.swagger_client.DebugApi
 
-All URIs are relative to *https://localhost/api/v1/*
+All URIs are relative to */api/v1/*
 
 Method | HTTP request | Description
 ------------- | ------------- | -------------
 [**post_generate_dockerfile**](DebugApi.md#post_generate_dockerfile) | **POST** /_debug/generate-dockerfile | Generate Dockerfile as it would be generated internally for inspections. 
 
-
 # **post_generate_dockerfile**
-> InspectionGenerateDockerfileResponse post_generate_dockerfile(specification)
+> InspectionGenerateDockerfileResponse post_generate_dockerfile(body)
```

Comment:
Does not look correct to me, will check this.

Reply:
Looks like its correctly referenced as `specification` correctly, the body is used by swagger-codegen.